snakefmt/formatter.py

```diff
@@ -28,6 +23,12 @@
 
 PathLike = Union[Path, str]
 rule_like_formatted = {"rule", "checkpoint"}
+valid_black_filemode_params = {
+    "target_versions",
+    "line_length",
+    "string_normalization",
+    "is_pyi",
+}
 
```

Comment:
We should not be doing this, because it is exposing details of `black`'s API and leading to us needing to track it: if they add or change a variable, we will also need to here. This should be removed.